spec/xcpretty/formatters/formatter_spec.rb

```diff
@@ -152,5 +156,15 @@ module XCPretty
 
 #{@formatter.red(SAMPLE_EXECUTED_TESTS)})
     end
+
+    it "show unmatched lines" do
+      @formatter = Formatter.new(true, true, true)
+      @formatter.format_other(SAMPLE_BUILD_SUCCEEDED).should == "unmatched: " + SAMPLE_BUILD_SUCCEEDED
```

Comment:
I think it would be better if you used the example you provided in the PR description here instead of `SAMPLE_BUILD_SUCCEEDED`.

Reply:
I was thinking of managing multiple destination tests ;-) so I've added an SAMPLE_UNMATCHED_LINE constant